libmscore/instrument.cpp

```diff
@@ -722,7 +739,7 @@ void Channel::read(XmlReader& e, Part* part)
                               ev.setChannel(0);
                               ev.setDataA(ctrl);
                               ev.setDataB(value);
-                              init.push_back(ev);
+                              initList().push_back(ev);
```

Comment:
Same as in constructor, direct usage of `_init` member may be more preferable here as the list doesn't seem to have an updated state here — and it is close to what was here before. But it seems both options would work.

Reply:
fixed